api/lib/config.js

```diff
@@ -28,14 +32,18 @@ var configPath = nconf.get('config')
 || process.env['TEST_CONFIG']
 || path.join(process.cwd(), 'config.json');
 
+if (!fs.existsSync(configPath)) {
```

Comment:
Yes. I'd also argue that we need another error to distinguish missing --config from missing config, but this can be done at a later time